Gruntfile.js

```diff
@@ -27,7 +28,8 @@ module.exports = function(grunt) {
             dist: {
                 files: {
                     'dist/js/bundle.js': ['js/index.js'],
-                    'dist/js/test.js': ['tests/js/web/index.js']
+                    'dist/js/test.js': ['tests/js/web/index.js'],
+                    'dist/js/lexicon-bundle.js': ['js/lexicon/lexicon-index.js']
```

Comment:
I think the bundle files should live in the dist directory, and not be committed to the repo.